File: qe-selenium-java/Activity12.java
package demos;

import java.time.Duration;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;
import org.openqa.selenium.support.ui.ExpectedConditions;
import org.openqa.selenium.support.ui.WebDriverWait;

public class Activity12 {
public static void main(String[] args) {
    WebDriver driver = new FirefoxDriver();
    WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(10));
    driver.get("https://training-support.net/webelements/dynamic-content");
    System.out.println(driver.getTitle());

    WebElement clickMeBtn = driver.findElement(By.id("genButton"));
    Actions actions = new Actions(driver);

    actions.click(clickMeBtn).perform();

    wait.until(ExpectedConditions.textToBe(By.id("word"), "release"));

    System.out.println(driver.findElement(By.id("word")).getText());

    driver.quit();

    

}
}

File: qe-selenium-python/Activity12.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC

with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/dynamic-content")
    wait = WebDriverWait(driver, 20)
    print(driver.title)
    actions = webdriver.ActionChains(driver)
    
    print("Title of the Page: "+driver.title)

    click_me_btn = driver.find_element(By.ID,"genButton")

    actions.click(click_me_btn).perform()

    wait.until(EC.text_to_be_present_in_element((By.ID, "word"), "release"))

    print(driver.find_element(By.ID,"word").text)
